docs/user/faq.rst

```diff
@@ -584,6 +585,43 @@ In the meantime, the workaround is to percent-encode the forward slash. If you
 don’t control the clients and can't enforce this, you can implement a Falcon
 middleware component to rewrite the path before it is routed.
 
+How do I adapt my code to default context type changes in Falcon 2.0?
+---------------------------------------------------------------------
+
+The default request/response context type has been changed from dictionary to a
+class subclassing dict in Falcon 2.0. Instead of setting dictionary items, one
```

Comment:
Suggested edit:

"one shall set attributes on the object now:" ➡️ "you can now simply set attributes on the object:"